app/konnect/service-catalog/integrations/github.md

```diff
@@ -0,0 +1,35 @@
+---
+title: GitHub Integration
+content-type: reference
+config:
+    discovery_support: true
+    discovery_default: true
+    bindable_entities: "Repositories"
+    mechanism: "pull/ingestion model"
+---
+
+_Type: External_
+
+The GitHub integration allows you to associate your Service Catalog service to one or more GitHub repositories. 
+
+For each linked Repository, the UI can show a **Repository Summary** with simple data pulled from the GitHub API, such as the number of open issues, open pull requests, most recently closed pull requests, languages, and more.
+
+## Authorize the GitHub integration
+
+1. From {{site.konnect_product_name}} select **[Integrations](https://cloud.konghq.com/us/service-catalog/integrations)**. 
+2. Select **GitHub**, then **Install Github**.
+3. Select **Authorize**. 
+
+This will take you to GitHub where you can grant {{site.konnect_product_name}} access to either **All Repositories** or **Select repositories**. 
+
+The {{site.konnect_product_name}} application can be managed from GitHub as a [GitHub Application](https://docs.github.com/en/apps/using-github-apps/authorizing-github-apps).
+
+## Bindable entities
+
+Entity | Binding Level | Description
```

Comment:
Note that we can remove the "Binding Level" column from github.md, traceable.md, and pagerduty.md. The "Binding Level" for all entities will only ever be "Service".

This was leftover stuff from back when we still had "Deployments" included as part of our object-model. Back then, a Resource could bind to either a Service or a Deployment. Now that Deployments are no longer a thing, a Resource can only ever bind to a Service.

Reply:
In case you are interested: https://docs.google.com/document/d/1AJ3Q0GsY_jt-ixpnrqjDvTrY5eDJR67uCmbVIwArzsc/edit#heading=h.kem0hwinnesa